tests/test_photonphase.py

```diff
@@ -127,6 +127,20 @@ def test_OrbPhase_column():
     hdul.close()
     os.remove(outfile)
 
+@pytest.mark.skipif(
+    "DISPLAY" not in os.environ, reason="Needs an X server, xvfb counts"
+)
+def test_nicer_result_bary_polyco(capsys):
+    "Check that barycentered NICER data is processed correctly with --polyco."
+    cmd = "--polycos {0} {1}".format(eventfile_nicer, parfile_nicer)
+    photonphase.main(cmd.split())
+    out, err = capsys.readouterr()
+    v = 0.0
+    for l in out.split("\n"):
+        if l.startswith("Htest"):
+            v = float(l.split()[2])
+    # Check that H-test is 216.67
+    assert abs(v - 216.67) < 1
```

Comment:
I can try and write a test that does that. Should I compare the first few phase values it calculates?

Reply:
Since numpy easily operates on vectors, I'd probably set some limit on `(A-B).std()` (if A and B are the arrays of phases computed with the two methods) so you are testing all the values not just a few.

Reply:
I have added the test, but am not certain what to set the limit to. It passes the test up to 1e-8.

Reply:
That is fantastic!